config/locales/modules/alpha.yml

```diff
@@ -124,16 +124,93 @@ en:
       1-2-1:
         heading: 1-2-1
         body: First page in second submodule
-      1-2-2:
-        heading: 1-2-2
+      1-2-1-1:
+        heading: 1-2-1-1
         body: |
           %Warning: people like stuff!%
-      1-2-3:
+      1-2-1-2:
         heading: 1-2-3
         body: Penultimate page
-      1-2-4:
+      1-2-2-3:
         heading: 1-2-4
         body: By viewing this page the training module is marked as complete
+
+      1-3:
+        heading: Summary and next steps
+        body: |
+          You have reached the end of this module's training content.
+
+          This next section of the module includes:
+
+          - a recap of what you have learned
+          - the end of module test
+          - an opportunity to reflect on your learning
+          - the chance to give us feedback about the module
+      
+      1-3-1:
+        heading: Recap
+        body: |
```

Comment:
The goal of our demo modules is to provide minimalist and descriptive content for use in testing and demoing, devoid of any confusing text about child development etc, instead containing whatever might provide clarity and disambiguation. Please can we strip out all the text and replace it with something else?